lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -2935,11 +2935,6 @@ getBankBitsFromStrings(const std::vector<std::string> &BitsStrings) {
 void addAnnotationDecorations(SPIRVEntry *E, DecorationsInfoVec &Decorations) {
   SPIRVModule *M = E->getModule();
   for (const auto &I : Decorations) {
-    // Such decoration already exists on a type, skip it
```

Comment:
A type (or a pointer variable) can have multiple annotations and this needs to be preserved during the LLVM to SPIR-V translation stage.